extensions/data-transfer/portability-data-transfer-imgur/src/main/java/org/datatransferproject/datatransfer/imgur/photos/ImgurPhotosExporter.java

```diff
@@ -67,6 +69,9 @@
   private final Monitor monitor;
   private final TemporaryPerJobDataStore jobStore;
 
+  // private URL url;
+  private UrlFactory urlFactory;
+
   public ImgurPhotosExporter(
```

Comment:
here you can just use constructor overloading (just call `this(params, here, all, passed, through,as, is, (url) -> new URL(url))`)

Reply:
changed that, thanks!